templates/ingress.yaml

```diff
@@ -2,7 +2,7 @@
 {{- $fullName := include "netdata.name" . -}}
 {{- $ingressPath := .Values.ingress.path -}}
 
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
The new API has only been [available since 1.14](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/) which is still quite recent (e.g. GKE stable as of today runs on 1.13). The old API is going away in 5 future versions from now (in 1.20). We should therefore delay this upgrade closer to removal of the API to keep as much backward compatibility as possible.